src/helper.jl

```diff
@@ -0,0 +1,436 @@
+                        Lib.XPRS_REPAIRINDEFINITEQ
+                        Lib.XPRS_MAXLOCALBACKTRACK
+                        Lib.XPRS_USERSOLHEURISTIC
+                        Lib.XPRS_FORCEPARALLELDUAL
+                        Lib.XPRS_BACKTRACKTIE
+                        Lib.XPRS_BRANCHDISJ
+                        Lib.XPRS_MIPFRACREDUCE
+                        Lib.XPRS_CONCURRENTTHREADS
+                        Lib.XPRS_MAXSCALEFACTOR
+                        Lib.XPRS_HEURTHREADS
+                        Lib.XPRS_THREADS
+                        Lib.XPRS_HEURBEFORELP
+                        Lib.XPRS_PREDOMROW
+                        Lib.XPRS_BRANCHSTRUCTURAL
+                        Lib.XPRS_QUADRATICUNSHIFT
+                        Lib.XPRS_BARPRESOLVEOPS
+                        Lib.XPRS_QSIMPLEXOPS
+                        Lib.XPRS_CONFLICTCUTS
+                        Lib.XPRS_PREPROTECTDUAL
+                        Lib.XPRS_CORESPERCPU
+                        Lib.XPRS_SLEEPONTHREADWAIT
+                        Lib.XPRS_PREDUPROW
+                        Lib.XPRS_CPUPLATFORM
+                        Lib.XPRS_BARALG
+                        Lib.XPRS_TREEPRESOLVE
+                        Lib.XPRS_TREEPRESOLVE_KEEPBASIS
+                        Lib.XPRS_TREEPRESOLVEOPS
+                        Lib.XPRS_LPLOGSTYLE
+                        Lib.XPRS_RANDOMSEED
+                        Lib.XPRS_TREEQCCUTS
+                        Lib.XPRS_PRELINDEP
+                        Lib.XPRS_DUALTHREADS
+                        Lib.XPRS_PREOBJCUTDETECT
+                        Lib.XPRS_PREBNDREDQUAD
+                        Lib.XPRS_PREBNDREDCONE
+                        Lib.XPRS_PRECOMPONENTS
+                        Lib.XPRS_MAXMIPTASKS
+                        Lib.XPRS_MIPTERMINATIONMETHOD
+                        Lib.XPRS_PRECONEDECOMP
+                        Lib.XPRS_HEURSEARCHROOTCUTFREQ
+                        Lib.XPRS_EXTRAELEMS
+                        Lib.XPRS_EXTRAPRESOLVE
+                        Lib.XPRS_EXTRASETELEMS
+                    ]
+
+
+n_variables(prob::XpressProblem) = getintattrib(prob, Lib.XPRS_COLS)
+n_constraints(prob::XpressProblem) = getintattrib(prob, Lib.XPRS_ROWS)
+n_special_ordered_sets(prob::XpressProblem) = getintattrib(prob, Lib.XPRS_SETS)
+n_quadratic_constraints(prob::XpressProblem) = getintattrib(prob, Lib.XPRS_QCONSTRAINTS)
+n_non_zero_elements(prob::XpressProblem) = getintattrib(prob, Lib.XPRS_ELEMS)
+n_quadratic_elements(prob::XpressProblem) = getintattrib(prob, Lib.XPRS_QELEMS)
+n_quadratic_row_coefficients(prob::XpressProblem) = getintattrib(prob, Lib.XPRS_QCELEMS)
+n_entities(prob::XpressProblem) = getintattrib(prob, Lib.XPRS_MIPENTS)
+n_setmembers(prob::XpressProblem) = getintattrib(prob, Lib.XPRS_SETMEMBERS)
+
+n_original_variables(prob::XpressProblem) = getintattrib(prob, Lib.XPRS_ORIGINALCOLS)
+n_original_constraints(prob::XpressProblem) = getintattrib(prob, Lib.XPRS_ORIGINALROWS)
```

Comment:
I picked names for these functions based on the Xpress documentation and what I thought would be most readable. I'm open to other suggestions or using what was previously used if people find this too verbose.

Reply:
those are fine, verbose is good